utils/build.ps1

```diff
@@ -1160,7 +1340,7 @@ function Build-LLVM([Platform]$Platform, $Arch) {
     -Platform $Platform `
     -UseMSVCCompilers C,CXX `
     -Defines @{
-      CMAKE_SYSTEM_NAME = if ($Platform -eq "Windows") { "Windows" } else { "Android" };
+      CMAKE_SYSTEM_NAME = $Platform.ToString();
```

Comment:
Is it the case that `CMAKE_SYSTEM_NAME` == `$Platform.ToString()` for all platforms?

Reply:
Yes, I believe so.